studio/data/entity-types/entity-type-query.ts

```diff
@@ -0,0 +1,109 @@
+import { UseQueryOptions } from '@tanstack/react-query'
+import { useCallback } from 'react'
+import { ExecuteSqlData, useExecuteSqlPrefetch, useExecuteSqlQuery } from '../sql/execute-sql-query'
+
+type EntityTypeArgs = {
+  id?: number
+}
+
+export const entityTypeSqlQuery = ({ id }: EntityTypeArgs) => {
+  const sql = /* SQL */ `
+    select
+      c.oid::int8 as "id",
+      nc.nspname as "schema",
+      c.relname as "name",
+      case c.relkind
+        when 'r' then 'table'
```

Comment:
alternatively we can just return the alphabets, and create an enum that maps these (ref data/database/database-query-constants.ts) - what do you think though?

Reply:
i'd say either way we still should have an enum actually - even if we convert the relkind to more readable strings here